cirq-core/cirq/vis/heatmap.py

```diff
@@ -299,6 +310,42 @@ def plot(
             ax = cast(plt.Axes, ax)
         original_config = copy.deepcopy(self._config)
         self.update_config(**kwargs)
+
+        highlighted_qubits = _as_set(kwargs.get("highlighted_qubits", ()))
+        if highlighted_qubits:
+            edgecolors = tuple(
+                (
+                    "red"
+                    if any(q in highlighted_qubits for q in qubits)
```

Comment:
Nit - frozenset has a method for this
```suggestion
                    if not highlighted_qubits.isdisjoint(qubits)
```